### ISSUE TITLE
The upload button needs a tabindex

### ISSUE THREAD

**Issue by marrus-sh** [marrus-sh/writer#4]
Right now it is not accessible via keyboard navigation.

**Comment by marrus-sh** [comment]
Fixed with 0.1.1 (0ec63509b768093ab2719203a246b066363c189d)
